tests/unit/test_image.py

```diff
@@ -107,3 +113,52 @@ def test_load_image_with_exif_info(self):
                 image_module.IMAGE_BACKEND.set(load_backend)
                 img = image_module.load_image(image_path)
                 assert img.shape == (10, 15, 3)
+
+
+class ImageDecodeTest:
+    @pytest.fixture
+    def fxt_img_four_channels(self) -> np.ndarray:
```

Comment:
```suggestion
    def fxt_img_four_channels(self) -> np.ndarray:
```
Could you extend the test to cover all 3 channel cases with 1, 3, and 4 channels? The practice showed it's important.

Reply:
done